### ISSUE TITLE
Wordpress link title does not work

### ISSUE THREAD

**Issue by joohub** [soflyy/breakdance-bugs#839]
Hello,
the link title does not get saved. The internal interlinking with the title tag is very important for onpage Seo and google rankings.  So i need a title tag for the Logo link for the Homepage and want to insert the main keyword. 
With the title tag i can add more value for google for what keyword the specific page should rank. 

If i add on my own page a non related keyword only into the title tag everywhere and other links, i will win rankings from nonranking to maybe page 60. So this title tag has big value in Seo / for google  

**What version of Breakdance are you using?**

Version 1.4.0-beta.1 

![link-title](https://github.com/soflyy/breakdance-bugs/assets/62914756/a3a3caa8-8264-42a5-87d7-dabd0d55120a)

**Comment by a-dubiel** [comment]
Are you using any extra plugins for link titles?

**Comment by joohub** [comment]
Yes, Iam using RankMath and the option to add automatic the title tag for images but nothing else. So it does not work everywhere, even on the footer links with the text link element. There is also no written "title" on the link. 
With Attributes it worked but it is a second todo i have todo for that everytime.

 b regards

**Comment by joohub** [comment]
I have a new fresh WP & breakdance install and i tested it again. I dont have an title tag option on a link. I can only entry the url. So this title tag ist needed. Should i add this everytime with Attributes ? The title tag is for onpage pretty important.
